src/dyninst/test1_20.C

```diff
@@ -134,14 +133,22 @@ test_results_t test1_20_Mutator::executeTest()
 	for (; blockIter != blocks.end(); blockIter++) 
 	{
 		BPatch_basicBlock *block = *blockIter;
-		assert(block);
+
+    if(!block) {
+     logerror("%s[%d]: Invalid block in 'func20_2'\n", __FILE__, __LINE__);
```

Comment:
This part probably needs to be indented correctly.

Reply:
Updated.